recipes/memsat-svm/reloc.patch

```diff
@@ -0,0 +1,22 @@
+--- run_memsat-svm.pl	2013-09-09 09:18:43.000000000 -0700
++++ run_memsat-svm.reloc.pl	2017-02-24 11:15:27.253626507 -0800
+@@ -22,16 +22,16 @@
+ 
+ ## IMPORTANT : this variable must be set here to the absolute path to the
+ ## library called "lib" and found in the same directory as this script.
+-use lib 'lib';
++use lib '/opt/anaconda1anaconda2anaconda3/share/memsat-svm/lib/';
```

Comment:
actually /opt/anaconda1anaconda2anaconda3/ is not a typo but the actual keyword for conda relocation mechanism